docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-setup.md

```diff
@@ -137,19 +131,44 @@ The **xCluster Disaster Recovery** tab also lists all the tables in replication
 
-- To delete a table from the replication, click **... > Remove Table**. This removes both the table and its index tables from replication. If you decide to remove an index table from the replication group, it does not remove its main table from the replication group.
+If YugabyteDB Anywhere is unable to obtain the status (for example, due to a heavy workload being run on the universe) the status for that table will be _Unable To Fetch_. You may refresh the page to retry gathering information.
 
-## Set up replication lag alerts
+| Status | Description |
+| :--- | :--- |
+| In Progress | The table is undergoing changes, such as being added to or removed from replication. |
+| Bootstrapping | The table is undergoing a full copy; that is, being backed up from the DR primary and being restored to the DR replica. |
+| Validated | The table passes pre-checks and is eligible to be added to replication. |
+| Operational | The table is being replicated. |
+
+The following statuses [trigger an alert](#set-up-replication-lag-alerts).
+
+| Status | Description |
+| :--- | :--- |
+| Failed | The table failed to be added to replication. |
+| Warning | The table is in replication, but the replication lag is more than the [maximum acceptable lag](#set-up-replication-lag-alerts), or the lag is not being reported. |
+| Dropped From Source | The table was in replication, but dropped from the DR primary without first being [removed from replication](../disaster-recovery-tables/#tables). If you are using Manual mode, you need to remove it manually from the configuration. In Semi-automatic mode, you don't need to remove it manually. |
+| Dropped From Target | The table was in replication, but was dropped from the DR replica without first being [removed from replication](../disaster-recovery-tables/#tables). If you are using Manual mode, you need to remove it manually from the configuration. In Semi-automatic mode, you don't need to remove it manually. |
+| Extra Table On Source | The table is newly created on the DR primary but is not in replication yet. |
+| Extra Table On Target | The table is newly created on the DR replica but it is not in replication yet. |
+| Error | The table is no longer replicating. This could be due to an issue such as a prolonged network partition. As a result, replication of the table can't continue, and you will need to [restart replication](#restart-replication). |
```

Comment:
This is shown in the UI only for old universes when the replication is broken and a restart is required.

For the new universes, the Error status will be replaced by the followings depending on the underlying reason causing the error:
Missing op ID
Schema mismatch
Missing table
Auto flag config mismatch

Considering the new statuses should work for universes >= 2.18, I think it also makes sense to remove the Error status.